environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/model/response/EnvironmentStatus.java

```diff
@@ -19,6 +19,8 @@
 
     CLUSTER_DEFINITION_CLEANUP_PROGRESS,
 
+    UMS_RESOURCE_DELETE_IN_PROGRESS,
```

Comment:
This requires an API update in the UI. Please mention here the UI pull request as well.